frontend/packages/shared/src/components/GiteaHeader/VersionControlButtons/components/ShareChangesPopover/CommitAndPushContent/FileChangesInfoModal/FilePath/FilePathUtils.test.ts

```diff
@@ -0,0 +1,50 @@
+import { convertPureGitDiffToUserFriendlyDiff } from 'app-shared/components/GiteaHeader/VersionControlButtons/components/ShareChangesPopover/CommitAndPushContent/FileChangesInfoModal/FilePath/FilePathUtils';
+
+const diffStringMock = `diff --git a/fileName.json b/fileName.json
+index 0909a03..527e226 100644
+--- a/fileName.json
++++ b/fileName.json
+@@ -2,6 +2,30 @@
+- old line
++ new line
+unchanged line
+\ No newline at end of file`;
+
+const diffStringWithoutLastMetadataLineMock = `diff --git a/fileName.json b/fileName.json
+index 0909a03..527e226 100644
+--- a/fileName.json
++++ b/fileName.json
+@@ -2,6 +2,30 @@
+- old line
++ new line
+unchanged line`;
```

Comment:
Skal det ikke være et linjeskift på slutten i dette tilfellet?
```suggestion
const diffStringWithoutLastMetadataLineMock = `diff --git a/fileName.json b/fileName.json
index 0909a03..527e226 100644
--- a/fileName.json
+++ b/fileName.json
@@ -2,6 +2,30 @@
- old line
+ new line
unchanged line
`;
```

Reply:
Usikker, men kan ta det med :D

Reply:
Jeg vil tro det er slik siden den andre diffen spesifiserer at det ikke er noe linjeskift på slutten. Men verifiser gjerne dette ved å se på hva endepunktet faktisk returnerer i det tilfellet. Og hvordan ser det ut når linjeskiftet på slutten enten er lagt til eller fjernet? Er det bare `+` eller `-` på en tom linje, eller kommer det en spesiell merknad da også?

Reply:
Vet ikke når det skjer. Eller om det kan skje. Er ikke sikkert det er en reell case